library/cc/key_value_store.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/common/pure.h"
+
+#include "absl/types/optional.h"
+#include "library/common/extensions/key_value/platform/c_types.h"
+
+namespace Envoy {
+namespace Platform {
+
+/**
+ *`KeyValueStore` is an interface that may be implemented to provide access to an arbitrary
```

Comment:
Nit, missing a space here:

```suggestion
 * `KeyValueStore` is an interface that may be implemented to provide access to an arbitrary
```

Reply:
Thanks, good catch.